cmd/metal-api/internal/service/partition-service.go

```diff
@@ -466,10 +466,9 @@ func (r *partitionResource) calcPartitionCapacity(pcr *v1.PartitionCapacityReque
 			size := sizesByID[cap.Size]
 
 			for _, reservation := range size.Reservations.ForPartition(pc.ID) {
-				reservation := reservation
-
 				cap.Reservations += reservation.Amount
 				cap.UsedReservations += min(len(machinesByProject[reservation.ProjectID].WithSize(size.ID).WithPartition(pc.ID)), reservation.Amount)
+				cap.Free -= cap.UsedReservations
```

Comment:
This is subject of discussion: Adds visibility to "overbooking" a certain size.

Reply:
Will this expose the free machines of a certain size ? if yes i would fill this field only for provider-tenants

Reply:
Moved into #538.